Harvest.Core/Domain/Expense.cs

```diff
@@ -62,9 +62,9 @@ internal static void OnModelCreating(ModelBuilder modelBuilder)
         {
             modelBuilder.Entity<Expense>().HasIndex(a => a.ProjectId);
 
+            modelBuilder.Entity<Expense>().Property(a => a.Price).HasPrecision(18, 2);
             modelBuilder.Entity<Expense>().Property(a => a.Total).HasPrecision(18, 2);
-
-            modelBuilder.Entity<Expense>().Property(a => a.Approved).HasDefaultValue(true);
+            modelBuilder.Entity<Expense>().Property(a => a.Quantity).HasPrecision(18, 2);
```

Comment:
I'm wondering about quantity. This doesn't have to be hours, it could be something where we want more precision

Reply:
@jSylvestre I think it's worth debating, I'm not sure the correct answer.  In the slack channel we just picked this since we use it for many other things.  However if we are going to go with 2 decimal places only we should probably enforce that client-side w/ validation too....